src/backend/TrafficCourts/Citizen.Service/Validators/DisputedCountValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentValidation;
+using TrafficCourts.Citizen.Service.Models.Dispute;
+
+namespace TrafficCourts.Citizen.Service.Validators
+{
+    /// <summary>
+    /// Fluent Validator for Disputed Count model
+    /// </summary>
+    public class DisputedCountValidator : AbstractValidator<DisputedCount>
+    {
+        public DisputedCountValidator()
+        {
+            RuleFor(_ => _.Plea).IsInEnum();
+            RuleFor(_ => _.Count).InclusiveBetween(1,3);
+
+            // Validation rules for properties if disputant pleaded guilty
+            RuleFor(_ => _.RequestTimeToPay).NotNull()
+                .When(_ => _.Plea.Equals(Plea.Guilty))
+                .WithMessage("'Request Time To Pay' selection is required to be set since disputant pleaded guilty");
+
+            RuleFor(_ => _.RequestReduction).NotNull()
+                .When(_ => _.Plea.Equals(Plea.Guilty))
+                .WithMessage("'Request Reduction' selection is required to be set since disputant pleaded guilty");
+
+            RuleFor(_ => _.AppearInCourt).NotNull()
+                .When(_ => _.Plea.Equals(Plea.Guilty))
+                .WithMessage("'Appear In Court' selection is required to be set since disputant pleaded guilty");
+
+            // Validation rules for properties if disputant pleaded not guilty
+            RuleFor(_ => _.RequestTimeToPay).Null()
+                .When(_ => _.Plea.Equals(Plea.NotGuilty))
+                .WithMessage("'Request Time To Pay' must not be set since disputant pleaded not guilty");
+
+            RuleFor(_ => _.RequestReduction).Null()
+                .When(_ => _.Plea.Equals(Plea.NotGuilty))
+                .WithMessage("'Request Reduction' must not be set since disputant pleaded not guilty");
+
+            RuleFor(_ => _.AppearInCourt).Null()
+                .When(_ => _.Plea.Equals(Plea.NotGuilty))
+                .WithMessage("'Appear In Court' must not be set since disputant pleaded not guilty");
+        }
+    }
+}
```

Comment:
So these rules match what the citizen-portal is doing in the UI?

Reply:
I'm not sure if the citizen-portal is exactly following this rule but it should be updated based on this if not. Ideally, it should not show the options to "Appear In Court" etc. if plea not guilty is selected. I will touch base with Shark about that.